src/server/deal.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
```

Comment:
I think any request we make to the server should from now on be in a server folder. This is super helpful because it abstracts it to a level where we can all access it and prevents duplicate code / siloing of network requests to the server